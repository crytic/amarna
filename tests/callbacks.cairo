// func shared_state_serialize{output_ptr: felt*}(shared_state: SharedState*) {
//     alloc_locals;
//     local output_start_ptr: felt* = output_ptr;
//     serialize_array(
//         array=program_output.asset_configs,
//         n_elms=program_output.n_asset_configs,
//         elm_size=AssetConfigHashEntry.SIZE,
//         callback=asset_config_hash_serialize + __pc__ - ret_pc_label,
//     );
//     serialize_word{output_ptr=output_start_ptr}(size);
//     return ();
// }
