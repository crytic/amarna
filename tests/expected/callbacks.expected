[arithmetic-add] in callbacks.cairo:8:18
[arithmetic-sub] in callbacks.cairo:8:18
[unused-arguments] in callbacks.cairo:1:49
[unused-function] in callbacks.cairo:1:1